src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -173,8 +173,8 @@ private boolean initSparks() {
     errors += SparkMaxUtils.check(elevatorRight.follow(elevatorLeft, true));
 
     //inverting stuff
-    errors += SparkMaxUtils.check(armEncoder.setInverted(true));
-    errors += SparkMaxUtils.check(elevatorLeftEncoder.setInverted(true));
+    errors += SparkMaxUtils.check(armAbsoluteEncoder.setInverted(true));
```

Comment:
We might *also* need to invert the relative encoders (the ones in the motors). Not sure. @balghane?

Reply:
I will await Basel's response for this.

Reply:
The inverting of the absolute encoder and the motor are separate. I believe the relative encoder gets inverted automatically when you invert the motor so it never needs to be inverted explicitly